engineering-handbook/the-wednesday-way/react-native.md

```diff
@@ -1,2 +1,341 @@
 # React-Native
 
+React-Native is fast becoming the go-to framework for mobile application development. Given the ability to create both Android and iOS applications using the same code base it is definitely one of our favourite frameworks for mobile app development.
```

Comment:
This is the perfect advert for react native.

I change it to this:
```
React-native is growing fast. We've been actively using the framework in production since the initial days. Here are a set of conventions we like to follow across all of our projects.```